src/os/histo/datebinmethod.js

```diff
@@ -222,6 +226,17 @@ os.histo.DateBinMethod.prototype.getValue = function(item) {
         // get the last hour of the year by getting the next new year and subtracting the previous new year
         var max = ((new Date(d.getUTCFullYear() + 1, 0) - min) / 86400000) * 24;
         return this.generateValues(t1, t2, max);
+      case os.histo.DateBinType.MONTH_OF_YEAR:
+        // 0-11
+        if (!d2)
+        {
```

Comment:
This is causing a lint error because curly braces should be on the same line as the statement. Could you fix this quick please?

Reply:
Sorry, this should be fixed now.